contracts/Removal.sol

```diff
@@ -155,6 +155,13 @@ contract Removal is
    */
   event ContractAddressesRegistered(IMarket market, ICertificate certificate);
 
+  /**
+   * @notice Emitted when the holdback percentage is updated for a project.
+   * @param projectId The ID of the project.
+   * @param holdbackPercentage The new holdback percentage for the project.
+   */
+  event HoldbackPercentageUpdated(uint256 projectId, uint8 holdbackPercentage);
+
```

Comment:
do you think we should also emit this event when the initial holdback percentage is set during `mintBatch` ?

Reply:
Nit: some of our events use past tense for event names, yet all of our inherited events use active tense as well as some of our own events. Id opt out for all events to be active tense since we can't change underlying event names. 

`UpdateHoldbackPercentage`

Reply:
Yes to your comment, but that should happen automatically if the initializer calls the function